File: DuDoong-Api/src/main/java/band/gosrock/api/event/model/dto/request/UpdateEventDetailRequest.java
package band.gosrock.api.event.model.dto.request;


import io.swagger.v3.oas.annotations.media.Schema;
import javax.validation.constraints.NotBlank;
import javax.validation.constraints.NotEmpty;
import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public class UpdateEventDetailRequest {
    // 포스터 이미지 url
    @Schema(defaultValue = "test/event/5/aa.jpeg", description = "포스터 이미지 key")
    @NotEmpty(message = "포스터 이미지 키 값을 입력해주세요")
    private String posterImageKey;

    // (마크다운) 공연 상세 내용
    @Schema(defaultValue = "고스락 공연에 여러분을 초대합니다 웰컴웰컴", description = "공연 상세 내용")
    @NotBlank(message = "공연장 상세 내용을 입력하세요")
    private String content;
}

File: DuDoong-Api/src/main/kotlin/band/gosrock/api/event/model/dto/request/UpdateEventDetailRequest.kt
package band.gosrock.api.event.model.dto.request

import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotEmpty

data class UpdateEventDetailRequest(
    // 포스터 이미지 url
    @field:Schema(defaultValue = "test/event/5/aa.jpeg", description = "포스터 이미지 key")
    @field:NotEmpty(message = "포스터 이미지 키 값을 입력해주세요")
    val posterImageKey: String? = null,

    // (마크다운) 공연 상세 내용
    @field:Schema(defaultValue = "고스락 공연에 여러분을 초대합니다 웰컴웰컴", description = "공연 상세 내용")
    @field:NotBlank(message = "공연장 상세 내용을 입력하세요")
    val content: String? = null
)
